daemon/protocol/python/scoot/daemon_cli.py

```diff
@@ -0,0 +1,156 @@
+"""
+Usage:
+  daemon_cli.py snapshot create <srcDir>
```

Comment:
I think we should consider making this the command-line tool named "scoot". Just scoot. It's how we expect users to use it...